### ISSUE TITLE
Styling bulleted lists & links on purple cards/jumbotrons

### ISSUE THREAD

**Issue by lcaple** [uweb/uw_wp_theme#25]
from Jill @ Pyramid:

- On purple cards and jumbotron overlays, the in-line hyperlink styling remains blue, instead of a white/reverse treatment, which makes it hard to read and not accessible. The two places this was most challenging was on purple full-width and large cards, which we found ourselves really gravitating toward using because of their styling. Not being able to have contextual hyperlinks in the copy on purple cards without doing some manual css work seems like it’ll present a challenge for other site editors, too. 

- On purple cards, bulleted lists are displayed in grey text, making them illegible. On all cards, the indent for bulleted lists is big enough that if you’re using them in 4 columns (3-up on a page), the line length of the text is quite short and not great for readability. Wondering if less of an indent on bullets might make them more friendly for use inside cards.

**Comment by lcaple** [comment]
We got this bug fixed in the latest release: https://github.com/uweb/uw_wp_theme/releases/tag/2.0.0
